boards/arm/stm32l496g_disco/pinmux.c

```diff
@@ -42,4 +42,4 @@ static int pinmux_stm32_init(struct device *port)
 }
 
 SYS_INIT(pinmux_stm32_init, PRE_KERNEL_1,
-	 CONFIG_PINMUX_STM32_DEVICE_INITIALIZATION_PRIORITY);
+		CONFIG_PINMUX_STM32_DEVICE_INITIALIZATION_PRIORITY);
```

Comment:
I think this change is not needed if you use 8 characters tabs as indicated in coding style guidelines:
http://docs.zephyrproject.org/contribute/contribute_guidelines.html#coding-style
then
https://www.kernel.org/doc/html/latest/process/coding-style.html#indentation,